groups/reconcile.go

```diff
@@ -214,19 +243,34 @@ func printGroupMembersAndSettings(srv *admin.Service, srv2 *groupssettings.Servi
 		log.Println("No groups found.")
 		return nil
 	}
+	log.Printf("groups:")
 	for _, g := range g.Groups {
-		// Don't touch existing mailing lists, we should
-		// always prefix with "k8s-infra-"
-		if !strings.HasPrefix(g.Email, "k8s-infra-") {
-			continue
+		//// Don't touch existing mailing lists, we should
```

Comment:
I don't understand this comment being here

Reply:
removing the comment and commented out code